app/src/test/java/com/sdp13epfl2021/projmag/database/FirebaseProjectsDatabaseTest.kt

```diff
@@ -170,8 +171,9 @@ class FirebaseProjectsDatabaseTest {
         Mockito.`when`(mockDS["tags"]).thenReturn(project.tags)
         Mockito.`when`(mockDS["isTaken"]).thenReturn(project.isTaken)
         Mockito.`when`(mockDS["description"]).thenReturn(project.description)
-        Mockito.`when`(mockDS["videoURI"]).thenReturn(project.videoURI)
+        Mockito.`when`(mockDS["videoUri"]).thenReturn(project.videoUri)
```

Comment:
This code is commented so there is no problem, but there is a typo : videoURI instead of videoUri.